src/libraries/System.Runtime.Serialization.Xml/tests/System.Runtime.Serialization.Xml.Tests.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <NoWarn>$(NoWarn);SYSLIB0021</NoWarn>
```

Comment:
I can't think of a reason why System.Runtime.Serialization.Xml.Tests would need the NoWarn.

If it's explicitly testing serialization for one of the obsoleted types, a pragma around that/those test(s) is more appropriate.  Otherwise, it would probably be a place that's doing the old/wrong pattern that should be updated to better code.

Reply:
The problem was [here](https://github.com/dotnet/runtime/pull/52303/checks?check_run_id=2511231508).
According to the errors the problematic lines are 
```csharp
static MD5CryptoServiceProvider md5 = null; // <--- line 211
```
and
```csharp
if (md5 == null)
  md5 = new MD5CryptoServiceProvider(); // <--- line 270
```
The method using this type:
https://github.com/dotnet/runtime/blob/1b30c00e5514aaa1e08b481582f0a49eea52e17f/src/libraries/System.Runtime.Serialization.Xml/tests/SerializationTestTypes/DataContract.cs#L267
What then needs to be done?

Reply:
Well, this code is just an example of the bad pattern.  So we should fix it :smile:

* Change the field type from MD5CryptoServiceProvider to MD5
* Change the ctor call to MD5.Create()

Reply:
I see, thanks for the instructions 🙂 Just fixed!